### ISSUE TITLE
[BUG] Could not install deb on ubuntu 21.10

### ISSUE THREAD

**Issue by vladimirshefer** [CCBlueX/LiquidBounce#758]
### LiquidBounce Build/Version

liquidlauncher_1.1.3_amd64.deb

### Operating System

Linux

### Minecraft Version

1.12

### Describe the bug

I was trying to install LiquidBounce Launcher, but installation failed instead of being successfully installed.

### Steps to reproduce

1. Download deb from download page (https://liquidbounce.net/download)
2. Try to install it with double click (via Ubunti Software)
3. Installation failed with message "Failed to install, file not supported"
![image](https://user-images.githubusercontent.com/23248545/163722085-ceff45e4-fbd8-4225-b1b2-35c89af25803.png)

Also i have tried to install it with `dpkg`, 
```
sudo dpkg -i liquidlauncher_1.1.3_amd64.deb 
```
but it says that package depends on `libgconf2-4`. This package is not found in neither default ubuntu repositories, not in `universe` resository. But i have found that there is `libgconf-2-4` (with `-` in the middle), but even after installing it the installation of LiquidBounce deb has failed.

### Client Log

```shell
> sudo dpkg -i liquidlauncher_1.1.3_amd64.deb 
Selecting previously unselected package liquidlauncher.
(Reading database ... 230802 files and directories currently installed.)
Preparing to unpack liquidlauncher_1.1.3_amd64.deb ...
Unpacking liquidlauncher (1.1.3-1) ...
dpkg: dependency problems prevent configuration of liquidlauncher:
 liquidlauncher depends on libgconf2-4; however:
  Package libgconf2-4 is not installed.

dpkg: error processing package liquidlauncher (--install):
 dependency problems - leaving unconfigured
Processing triggers for mailcap (3.69ubuntu1) ...
Processing triggers for desktop-file-utils (0.26-1ubuntu2) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu1) ...
Errors were encountered while processing:
 liquidlauncher
```

**Comment by 1zun4** [comment]
Yes. Sorry. This is a known issue. Do you mind if you try installing it manually? The current launcher is discontinued and somehow the libgconf2-4 got renamed into libgconf-2-4 or something, which now causes issues on install. Not sure if we can fix this directly in the deb. 
Anyway, the launcher will be replaced by this rewrite soon: https://github.com/CCBlueX/LiquidLauncher
